### DISCUSSION TITLE
How to relabel columns that start with numbers or contain spaces?

### DISCUSSION THREAD

**Discussion by micycle1** [posit-dev/great-tables#355] [Q&A] [answered]
How can one do this?

Renaming columns with `.cols_label()` can be problematic because the column names have to be provided as **keyword arguments**. This means that one cannot relabel columns that start with numbers or contain spaces, etc because they aren't valid keywords in Python.

**Comment by jrycw** [comment, marked as answer]
@micycle1, you might want to try something like this:
```python
import pandas as pd
from great_tables import GT

df = pd.DataFrame({" a": [1, 2], "2b": [3, 4]})
GT(df).cols_label(**{" a": "a", "2b": "b"})
```

**Comment by jrycw** [comment]
It seems there are no follow-up issues related to this discussion, so I'll close it for now.
